pkg/kudoctl/cmd/init_integration_test.go

```diff
@@ -41,6 +44,86 @@ func TestMain(m *testing.M) {
 	os.Exit(exitCode)
 }
 
+const (
+	operatorFileName        = "kudo_v1alpha1_operator.yaml"
+	operatorVersionFileName = "kudo_v1alpha1_operatorversion.yaml"
+	instanceFileName        = "kudo_v1alpha1_instance.yaml"
+	manifestsDir            = "../../../config/crds/"
+)
+
+func TestCrds_Config(t *testing.T) {
+	crds := cmdinit.CRDs()
+
+	if false {
+		// change this to true if you want to one time override the manifests with new values
+		// this should be used only when the manifests changed in your PR and you want to update to the newly generated values
+		err := writeManifest(operatorFileName, crds.Operator)
+		if err != nil {
+			t.Errorf("Operator file override failed: %v", err)
+		}
+		err = writeManifest(operatorVersionFileName, crds.OperatorVersion)
+		if err != nil {
+			t.Errorf("OperatorVersion file override failed: %v", err)
+		}
+		if err != nil {
+			t.Errorf("Instance file override failed: %v", err)
+		}
+		err = writeManifest(instanceFileName, crds.Instance)
+	}
+
+	err := verifyManifestFile(operatorFileName, crds.Operator)
+	if err != nil {
+		t.Errorf("Operator file verification failed: %v", err)
+	}
+	err = verifyManifestFile(operatorVersionFileName, crds.OperatorVersion)
+	if err != nil {
+		t.Errorf("OperatorVersion file verification failed: %v", err)
+	}
+	err = verifyManifestFile(instanceFileName, crds.Instance)
+	if err != nil {
+		t.Errorf("Instance file verification failed: %v", err)
+	}
+}
+
+func writeManifest(fileName string, expectedObject runtime.Object) error {
+	expectedContent, err := runtimeObjectAsBytes(expectedObject)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Updating file %s", fileName)
+	path := filepath.Join(manifestsDir, fileName)
+	if err := ioutil.WriteFile(path, expectedContent, 0644); err != nil {
+		return fmt.Errorf("failed to update config file: %s", err)
+	}
+	return nil
+}
+
+func verifyManifestFile(fileName string, expectedObject runtime.Object) error {
+	expectedContent, err := runtimeObjectAsBytes(expectedObject)
+	if err != nil {
+		return err
+	}
+	path := filepath.Join(manifestsDir, fileName)
+	of, err := ioutil.ReadFile(path)
+	if err != nil {
+		return fmt.Errorf("failed reading manifest file %s: %s", fileName, err)
+	}
+
+	if !bytes.Equal(expectedContent, of) {
+		return fmt.Errorf("%s does not match output of init %s", fileName, string(of))
+	}
```

Comment:
Can we just compare string here, please? `assert.Equal` also gives you the diff which makes debugging the test so much easier:
```suggestion
	assert.Equal(t, string(expectedContent), string(of), "manifest file does not match the existing one")
```
And you don't need `runtimeObjectAsBytes` then too.